modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -91,10 +91,7 @@
             <class name="org.wso2.identity.integration.test.requestPathAuthenticator.RequestPathBasicAuthenticationSSOTest" />
             <class name="org.wso2.identity.integration.test.idp.mgt.IdentityProviderMgtServiceTestCase" />
             <class name="org.wso2.identity.integration.test.idp.mgt.ResidentIDPConfigsTestCase" />
-
-            <!--Temporarily disabling until the test is resolved.
             <class name="org.wso2.identity.integration.test.idp.mgt.PreferenceAPIIntegrationUITestCase" />
```

Comment:
There is block of tests that change the config updates. Move the test to that block.

Reply:
Ack

Reply:
fixed with https://github.com/wso2/product-is/pull/19182/commits/569c8adc157c129fb7bf0cd02ca248189cc65577